diksha/diksha.init.js

```diff
@@ -76,41 +76,29 @@ let processEcarFiles = (filePath) => {
     Adds the JSON files to BleveSearch Database
 */
 
-let jsonDocsToDb = (dir) => {
-    let defer = q.defer();
+let jsonDirTob = () => {
```

Comment:
Give this a better name, e.g `indexMetaDataIntoBleveDb`

Reply:
Done